ngi_pipeline/engines/piper_ngi/launchers.py

```diff
@@ -392,6 +392,17 @@ def sbatch_piper_sample(command_line_list, workflow_name, project, sample,
                                                    project_id=project.project_id,
                                                    sample_id=sample.name)
     sbatch_text_list.append("\nPIPER_RETURN_CODE=$?")
+    #Precalcuate md5sums
+    sbatch_text_list.append('MD5FILES="$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.bam'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.bai'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/05_processed_alignments/*.metrics'.format(project.project_id))
+    sbatch_text_list.append('$SNIC_TMP/ANALYSIS/{}/piper_ngi/07_variant_calls/*.genomic.vcf'.format(project.project_id))
```

Comment:
.bai and .metrics are going to be md5'ed anyway, right ?
It makes sense to me to have all md5 calculated on scratch, as if the copying back from scratch messes up, then the md5 checks will fail as expected.
Maybe I'm missing another case though ?

.genomic.vcf should indeed be changed to .genomic.vcf.gz